python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -91,16 +91,6 @@ def __str__(self):
         return "JobError: %s" % str(self.what)
 
 
-class PreparedStateError(GangaException):
```

Comment:
Can we move this to somewhere closes to the Prep repository as it should be defined there imo or in Core.exceptions